src/arc-validate/arc-validate.fsproj

```diff
@@ -16,37 +14,34 @@
     <Compile Include="Commands\PackageCommand.fs" />
     <Compile Include="Commands\ARCValidateCommand.fs" />
     <Compile Include="APIs\PackageAPI.fs" />
+    <Compile Include="APIs/BaselineValidation.fs" />
     <Compile Include="APIs\ValidateAPI.fs" />
     <Compile Include="APIs\ARCValidateAPI.fs" />
     <Compile Include="CommandHandling.fs" />
-    <Compile Include="Program.fs" />    
+    <Compile Include="Program.fs" />
   </ItemGroup>
-
   <ItemGroup>
     <PackageReference Include="Argu" Version="[6.1.4]" />
     <PackageReference Include="Expecto" Version="[9.0.4]" />
     <PackageReference Include="Spectre.Console" Version="0.48.0" />
     <PackageReference Include="Spectre.Console.CLI" Version="0.48.0" />
   </ItemGroup>
-
   <ItemGroup>
     <ProjectReference Include="..\ARCExpect\ARCExpect.fsproj" PackageVersion="[1.0.0, 2.0.0)" />
     <ProjectReference Include="..\ARCValidationPackages\ARCValidationPackages.fsproj" PackageVersion="[1.0.0, 2.0.0)" />
   </ItemGroup>
-
   <Target Name="UseExplicitPackageVersions" BeforeTargets="GenerateNuspec">
     <ItemGroup>
-      <_ProjectReferenceWithExplicitPackageVersion Include="@(ProjectReference->'%(FullPath)')" Condition="'%(ProjectReference.PackageVersion)' != ''" />
-      <_ProjectReferenceWithExactPackageVersion Include="@(ProjectReference->'%(FullPath)')" Condition="'%(ProjectReference.ExactVersion)' == 'true'" />
+      <_ProjectReferenceWithExplicitPackageVersion Include="@(ProjectReference-&gt;'%(FullPath)')" Condition="'%(ProjectReference.PackageVersion)' != ''" />
+      <_ProjectReferenceWithExactPackageVersion Include="@(ProjectReference-&gt;'%(FullPath)')" Condition="'%(ProjectReference.ExactVersion)' == 'true'" />
       <_ProjectReferenceWithReassignedVersion Include="@(_ProjectReferencesWithVersions)" Condition="'%(Identity)' != '' And '@(_ProjectReferenceWithExplicitPackageVersion)' == '@(_ProjectReferencesWithVersions)'">
-        <ProjectVersion>@(_ProjectReferenceWithExplicitPackageVersion->'%(PackageVersion)')</ProjectVersion>
+        <ProjectVersion>@(_ProjectReferenceWithExplicitPackageVersion-&gt;'%(PackageVersion)')</ProjectVersion>
```

Comment:
Is there a need for replacing these characters? It still builds locally, but i would prefer leaving this untouched if there is no specific need